src/System.Management.Automation/engine/remoting/commands/StartJob.cs

```diff
@@ -480,6 +480,25 @@ public virtual ScriptBlock InitializationScript
 
         private ScriptBlock _initScript;
 
+
+        /// <summary>
+        /// Working directory of the process.
+        /// </summary>
+        [Parameter]
+        [ValidateNotNullOrEmpty]
+        public override string WorkingDirectory
+        { 
+            get
+            {
+                return base.WorkingDirectory;
+            }
+
+            set
+            {
+                base.WorkingDirectory = value;
+            }
```

Comment:
This can be simplified to:
```csharp
get => base.WorkingDirectory;
set => base.WorkingDirectory = value;
```

Reply:
No strong opinion here but: I am tempted to leave this as it is and the reason for that is the rest of the file that I am touching follows this pattern for the get\sets [source](https://github.com/davinci26/PowerShell/blob/837eb6a5b6a389a080b5685ea87a27bbb0021169/src/System.Management.Automation/engine/remoting/commands/StartJob.cs)

Reply:
Generally speaking from what I've seen, when most contributors add new code, they do so using the current practices rather than match existing syntax. This keeps us forward moving. Eventually PRs come in to advance old syntax to catch up to newer syntax.

Reply:
Sure, I will fix it in the next iteration